subsys/bluetooth/host/gatt.c

```
@@ -5442,7 +5464,7 @@ static void add_subscriptions(struct bt_conn *conn)
 			/* Force write to CCC to workaround devices that don't
 			 * track it properly.
 			 */
-			gatt_write_ccc(conn, params);
+			gatt_resub_ccc(conn, params);
```

May be outside the scope of this PR, but if this fails, it fails silently :|